src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,61 @@
+- The ninja visor can now alternate between normal vision and night vision.
+
+### Objective Mechanics
+
+- All stealing objectives can now only be performed with a specialized item that is dropped by the ninja upon death, unlike previously where they only used their gloves - This allows the crew to regain their losses.
+- A strongbox for Cargo containing all their deposited spesos, can be the target of one of the Space Ninja's objectives, located in the vault - An alternative to "Steal X technologies".
+- An AI law board that can be used for one of the Space Ninja's objectives - An alternative to "Detonate your spider charge".
+- The criminal records computer now only announces the error locally in the Security channel, instead of it being broadcasted as a Central Command announcement station-wide.
+- The communications computer objective now revolves around the Space Ninja sending a Spider Clan Announcement from it, the contents of said announcement being up to their discretion. Life before, it requires the use of turned on space ninja gloves, the one key difference being that now the EMAG interaction is instantaneous, overriding all access restrictions as well as unlocks Spider Clan as an announcement sender.
+
+## Balance Changes
+
+### Nerfs
+- [Fix ninja stunlocking](https://github.com/space-wizards/space-station-14/pull/33244)
+- All Space Ninjas are now bio-engineered humans, making it so they don't get specie-specific attributes such as tail-dragging.
+- The space ninja gloves have lost their EMAG interactions with lathes and printers.
+- Removes the survival box from the ninja's satchel, as there's no longer any use for it.
+- Lower the energy katana's one-handed damage.
+- Lower the ninja throwing star's piercing damage to compensate for the newly added ninja caltrops.
+- Lower the ninja throwing star's stamina damage.
+
+### Buffs
+- Space Ninjas no longer have a need to eat nor drink.
+- Space Ninjas now come outfitted with a ninja headset as per [Alwayswannahunt's PR](https://github.com/space-wizards/space-station-14/pull/32841) , green encryption key included.
+- To compensate for the newly removed EMAG interactions with lathes and printers, the space ninja gloves can now unlock access-restricted lockers and crates.
+- The katana dash is considerably quieter, at the cost of a longer recharge duration.
+- The energy katana's slash rate is slightly faster than it was previously when one-handed to compensate for it's damage loss.
+- Grants the ability to wield the energy katana for a hard-hitting swing, at the cost of a slower slash rate.
+- The ninja throwing stars and ninja caltrops can be stacked, as well as thrown in spreads.
+- When phased, the Space Ninja is able to discreetly steal from people, at the cost of a longer action duration.
```

Comment:
> I'd like to see the ninja's phase cloak be significantly buffed to near-invisibility when cloaked given the ninja is moving _stupidly slow_ in order to better enable this interaction.

> I feel like giving ninjas their cloak similar to how stealth boxes behave could be something interesting to look at, but have it be tuned to reflect their role better. Speedy ninja = not as cloaked, slow and steady ninja = very cloaked, extremely hard to see.

same here! I think it'd be super cool if the ninja had a system similar to that of loneops' cardboard boxes, just a tad afraid of how meta it could become for the ninja to just walk _super_ slowly everywhere

Reply:
Ninja lives and dies off of speed. Part of the ninja's survivability is the ability to get in and out of basically any situation lightning quick before someone can even _radio_ security, whether they are cloaked or not.

The proposed change can put players in a different mindset:

Do I slowly sneak up to captain and copy their ID, or steal their bag, concealing myself to make my presence unaware to security, or do I go in loud and make a mess, allowing security to know that I exist and cause them to gear up?

There's a lot of different ways you could play this.

Reply:
> Do I slowly sneak up to captain and copy their ID, or steal their bag, concealing myself to make my presence unaware to security, or do I go in loud and make a mess, allowing security to know that I exist and cause them to gear up?

this sold me on it, sounds super fun to be able to change your strategy depending on the go with something simple like this too, I'll put it in in a bit👍